app/controllers/filters_controller.rb

```diff
@@ -12,7 +12,11 @@ def index
   end
 
   def new
-    @filter = resource_base.build
+    render 'react/index'
+  end
+
+  def edit
+    render 'react/index'
```

Comment:
Should it be done in the `routes.rb` instead?

Reply:
yup